.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Docker image (HEAD)
+on:
+  push:
+    branches: [master]
+    tags: ['v*.*.*']
+  pull_request:
```

Comment:
```suggestion

```

Reply:
We shouldn't run this on PR because it requires secrets. They won't be available to external contributors and this CI run will fail which will be a negative experience for those folks.